Docs/User_Guides/WebUI_Extension/TTS-SETUP-GUIDE.md

```
@@ -0,0 +1,724 @@
+# TTS Provider Setup Guide
+
+This guide explains how to set up each TTS provider, especially the local models like Higgs, Kokoro, and VibeVoice.
+
+## Table of Contents
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)